codegen/templates/main.tmpl

```diff
@@ -29,18 +30,23 @@ func getConfigDirName() string {
 	)
 }
 
-func getConfig() *zanzibar.StaticConfig {
+func getConfig(seedConfig map[string]interface{}) *zanzibar.StaticConfig {
 	return zanzibar.NewStaticConfigOrDie([]string{
 		filepath.Join(getDirName(), "zanzibar-defaults.json"),
 		filepath.Join(getConfigDirName(), "production.json"),
 		filepath.Join(os.Getenv("CONFIG_DIR"), "production.json"),
-	}, nil)
+	}, seedConfig)
 }
 
 func createGateway() (*zanzibar.Gateway, error) {
-	config := getConfig()
-	
-	gateway, err := zanzibar.CreateGateway(config, nil)
+    listenIP, err := tchannel.ListenIP()
```

Comment:
Can we move this logic into `gateway.Bootstrap()` instead of being in generated code ?